### ISSUE TITLE
Can not zoom in and out the structure

### ISSUE THREAD

**Issue by superstar54** [aiidalab/aiidalab-widgets-base#362]
## System Information
latest version of 'aiidalab/aiidalab-docker-stack:latest'

## Exact steps for others to reproduce the behavior
- Open aiidalab-qe
- In step 1, select `Silicon diamond` from the examples.
- Try to zoom in and zoom out

## Expected results
Zoom in and out do not work.

**Comment by yakutovicha** [comment]
@superstar54 are you sure this does not work? For me, it is enough to keep the right button clicked and move the mouse.

**Comment by superstar54** [comment]
@yakutovicha Thanks! It works. May I ask what is the button for translation?
Maybe we can add a tooltip to help people find the right combination of the button and mouse.

**Comment by yakutovicha** [comment]
> @yakutovicha Thanks! It works. May I ask what is the button for translation?

Yes, if you have a mouse, you need to _click_ on the wheel and move the mouse.

> Maybe we can add a tooltip to help people find the right combination of the button and mouse.

Sure. Can you take care of this?

**Comment by superstar54** [comment]
Thanks. I will add the tooltip.
